pkg/interface/src/apps/chat/components/lib/content/url.js

```diff
@@ -79,11 +80,19 @@ export default class UrlContent extends Component {
           >
             {content.url}
           </a>
-          <a className="bs ml2 f7 pointer lh-copy v-top"
-             onClick={e => this.unfoldEmbed()}
+          <Box
+            border={1}
+            borderRadius={2}
+            display='inline-block'
```

Comment:
```suggestion
            display='inline-block'
            fontSize={0}
```

<img width="600" alt="image" src="https://user-images.githubusercontent.com/20846414/90286000-ecc89e80-de42-11ea-8b66-84b69be50c7e.png">

Reply:
much better